Tools/Scripts/webkitpy/benchmark_runner/data/plans/motionmark1.3-30fps.plan

```diff
@@ -0,0 +1,252 @@
+{
+    "timeout": 1800,
+    "count": 3,
+    "github_source": "https://github.com/webkit/MotionMark/tree/5d9c88136d59c11daf78d539c73e4e3e88c091ab/MotionMark",
+    "webserver_benchmark_patch": "data/patches/webserver/MotionMark1.3.patch",
+    "webdriver_benchmark_patch": "data/patches/webdriver/MotionMark1.3.patch",
+    "signpost_patch": "data/patches/signposts/MotionMark1.3.patch",
+    "entry_point": "index.html",
+    "subtest_entry_point": "developer.html",
+    "config": {
+        "orientation": "landscape"
+    },
+    "output_file": "motionmark.result",
+    "subtest_url_format": "suite-name=${SUITE}&test-name=${TEST}&frame-rate=30&system-frame-rate=30",
```

Comment:
It is really unfortunate that you copied the whole file to change just one line. Can't we do this by passing a new argument to `run-benchmark`?

Reply:
Sorry, I was in the need for a quick test... We are looking into a proper solution and will revert that soon. Ok?